content-services/latest/install/containers/docker-compose.md

```diff
@@ -266,41 +257,24 @@ Use this information to verify that the system started correctly, and to clean u
-        Removing acs-trial_transform-router_1   ... done
-        Removing acs-trial_transform-core-aio_1 ... done
-        Removing acs-trial_proxy_1              ... done
-        Removing acs-trial_postgres_1           ... done
-        Removing acs-trial_alfresco_1           ... done
-        Removing acs-trial_digital-workspace_1  ... done
-        Removing acs-trial_solr6_1              ... done
-        Removing acs-trial_share_1              ... done
-        Removing acs-trial_control-center_1     ... done
-        Removing acs-trial_sync-service_1       ... done
-        Removing acs-trial_shared-file-store_1  ... done
-        Removing acs-trial_activemq_1           ... done
-        Removing network acs-trial_default
-        Removing image quay.io/alfresco/alfresco-content-repository:7.4.0.1
-        Removing image quay.io/alfresco/alfresco-shared-file-store:2.1.0
-        Removing image quay.io/alfresco/alfresco-share:7.4.0.1
-        Removing image postgres:14.4
-        Removing image quay.io/alfresco/search-services:2.0.7
-        Removing image alfresco/alfresco-activemq:5.17.1-jre11-rockylinux8
-        Removing image alfresco/alfresco-transform-core-aio:3.1.0
-        Removing image quay.io/alfresco/alfresco-transform-router:2.1.0
-        Removing image quay.io/alfresco/alfresco-digital-workspace:4.0.0
-        Removing image quay.io/alfresco/alfresco-control-center:8.1.0
-        Removing image alfresco/alfresco-acs-nginx:3.4.2
-        Removing image quay.io/alfresco/service-sync:3.9.0
+        Container acs_trial-transform-core-aio-1                           Removed
+        Container acs_trial-transform-router-1                             Removed
+        Container acs_trial-search-reindexing-1                            Removed
+        Container acs_trial-proxy-1                                        Removed
+        Container acs_trial-search-1                                       Removed
+        Container acs_trial-sync-service-1                                 Removed
+        Container acs_trial-postgres-1                                     Removed
+        Container acs_trial-elasticsearch-1                                Removed
+        Container acs_trial-activemq-1                                     Removed
+        Container acs_trial-share-1                                        Removed
+        Container acs_trial-digital-workspace-1                            Removed
+        Container acs_trial-alfresco-1                                     Removed
+        Container acs_trial-control-center-1                               Removed
+        Image quay.io/alfresco/service-sync:4.0.1                          Removed
+        ...                               
+        Image quay.io/alfresco/alfresco-content-repository:23.2.1          Stopping                               
+        ...
+        Network acstrial_default                                           Stopping
```

Comment:
Looks great @mbujak-hyland! I think these need to be written consistently (unless the console output has changed):

- The folder name for testing is defined as `acs_trial`.
- However, `acstrial_*` differs in various places, such as `acstrial_*` (line 277), `acs_trial-*` (lines 260-272), and `acs-trial-*` (other examples).

Reply:
Thank you for that, @anxumalo! I fixed it to include only the `acs_trial` folder name.